constants.xml

```diff
@@ -22,19 +22,18 @@
         <NUM_DS_KEEP_TX_BODY>5</NUM_DS_KEEP_TX_BODY>
         <MAXMESSAGE>800</MAXMESSAGE>
         <MAXSUBMITTXNPERNODE>10</MAXSUBMITTXNPERNODE>
-        <N_PREFILLED_PER_ACCOUNT>1000000</N_PREFILLED_PER_ACCOUNT>
+        <MICROBLOCK_GAS_LIMIT>500</MICROBLOCK_GAS_LIMIT>
         <NEW_NODE_POW_DELAY>10</NEW_NODE_POW_DELAY>
         <POST_VIEWCHANGE_BUFFER>10</POST_VIEWCHANGE_BUFFER>
         <CONTRACT_CREATE_GAS>50</CONTRACT_CREATE_GAS>
         <CONTRACT_INVOKE_GAS>10</CONTRACT_INVOKE_GAS>
         <NORMAL_TRAN_GAS>1</NORMAL_TRAN_GAS>
         <WAITING_FORWARD>5</WAITING_FORWARD>
         <COINBASE_REWARD>100</COINBASE_REWARD>
-        <TXN_SUBMISSION>5</TXN_SUBMISSION>
-        <TXN_BROADCAST>10</TXN_BROADCAST>
         <DEBUG_LEVEL>3</DEBUG_LEVEL>
         <BROADCAST_INTERVAL>60</BROADCAST_INTERVAL>
         <BROADCAST_EXPIRY>600</BROADCAST_EXPIRY>
+        <TX_DISTRIBUTE_TIME_IN_MS>10000</TX_DISTRIBUTE_TIME_IN_MS>
```

Comment:
For consistency with other constants, I suggest to use seconds instead of ms

Reply:
If don't use this sleep, the time between two sleeps will be ms level. So used ms for better tuning